packages/@ama-sdk/schematics/schematics/migrate/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Rule, type Tree } from '@angular-devkit/schematics';
+import { MigrateSchematicsSchemaOptions } from './schema';
+import { getMigrationRuleRunner, type MigrationRulesMap } from '@o3r/schematics';
+import { resolve } from 'node:path';
+
+const tsMigrationMap: MigrationRulesMap = {
+
+};
+
+const isTypescriptSdk = (tree: Tree) => {
+  return tree.exists('/tsconfig.json');
+};
+
+/**
+ * Generate a Extension of a API core definition
+ * @param options
+ */
+function migrateFn(options: MigrateSchematicsSchemaOptions): Rule {
+
+  const to: string = options.to || JSON.parse(require(resolve(__dirname, '..', '..', 'package.json'))).version;
+
+  return (tree, context) => {
+    const runMigrateSchematic = isTypescriptSdk(tree) ? getMigrationRuleRunner(tsMigrationMap, { logger: context.logger }) : undefined;
```

Comment:
I don;t really understand how it works, as the tsMIgrationMap is empty. when is it supposed to be filled ?

Reply:
It will be filled by the 2 following Pull Requests:
- #1744
- #1696